src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java

```diff
@@ -811,11 +811,11 @@ protected void addOneAuditLog(AuditLogBean audit, String currentIndent) {
                 }
                 if (o.length() > 0) {
                     xml.append(nls);
-                    xml.append(currentIndent + "                      OldValue=\"" + org.apache.commons.lang3.StringEscapeUtils.escapeXml11(o) + "\" ");
+                    xml.append(currentIndent + "                      OldValue=\"" + StringEscapeUtils.escapeXml(o) + "\" ");
                 }
                 if (n.length() > 0) {
                     xml.append(nls);
-                    xml.append(currentIndent + "                      NewValue=\"" + org.apache.commons.lang3.StringEscapeUtils.escapeXml11(n) + "\"");
+                    xml.append(currentIndent + "                      NewValue=\"" + StringEscapeUtils.escapeXml(n) + "\"");
```

Comment:
looks like you reverted Suthirr's changes when resolving merge conflicts. Can you change it back?

Reply:
all set , reverted back